src/main/java/com/imsweb/layout/record/fixed/FixedColumnsLayout.java

```diff
@@ -532,6 +522,10 @@ public void verify() {
                 if (f1.getEnd() >= f2.getStart())
                     throw new RuntimeException("Fields " + f1.getName() + " and " + f2.getName() + " are overlapping");
 
+                //verify there are no gaps for v16 or later (gaps were replaced with reserved fields)
```

Comment:
You can't do this here. It's perfectly alright for a fixed-column layout to have gaps. Only NAACCR layouts shouldn't have gaps. Please remove this check and replace it with a unit test that runs in each version of NAACCR layout test.